autogtp/main.cpp

```diff
@@ -76,39 +85,13 @@ int main(int argc, char *argv[]) {
     if (gpusNum == 0) {
         gpusNum = 1;
     }
+    if(parser.isSet(singleOption)) {
 
-    // Map streams
-    QTextStream cin(stdin, QIODevice::ReadOnly);
-    QTextStream cout(stdout, QIODevice::WriteOnly);
-#if defined(LOG_ERRORS_TO_FILE)
-    // Log stderr to file
-    QFile caFile("output.txt");
-    caFile.open(QIODevice::WriteOnly | QIODevice::Text | QIODevice::Append);
-    if(!caFile.isOpen()){
-        qDebug() << "- Error, unable to open" << "outputFilename" << "for output";
+        gamesNum = 1;
+        gpusNum = 1;
     }
-    QTextStream cerr(&caFile);
-#else
+    // Map streams
     QTextStream cerr(stderr, QIODevice::WriteOnly);
-#endif
-#ifdef WIN32
-    // We need to make sure these files we need are there before calling them.
-    // Otherwise it will result in a crash.
```

Comment:
This check was nice to have I think?

Reply:
I guess they disappeared because of a wrong rebase. I will check which and fix it when I get home